src/script/backup/CrossPlatformBackup/data.schema.ts

```diff
@@ -19,55 +19,55 @@
 
 import zod from 'zod';
 
-const ConversationSchema = zod.object({
-  accessModes: zod.array(zod.string()).optional(),
-  accessRole: zod.array(zod.any()).optional(),
-  accessRoleV2: zod.string().optional(),
-  archived_state: zod.boolean(),
-  archived_timestamp: zod.number(),
-  cipher_suite: zod.number().optional(),
-  creator: zod.string(),
+// For now we just check for the data we urgently need for the backup
+export const ConversationTableEntrySchema = zod.object({
   domain: zod.string(),
-  group_id: zod.string().optional(),
   id: zod.string(),
-  last_event_timestamp: zod.number(),
-  last_server_timestamp: zod.number(),
-  message_timer: zod.string().nullable().optional(),
-  muted_state: zod.number().nullable(),
-  muted_timestamp: zod.number(),
-  name: zod.string().nullable(),
-  others: zod.array(zod.any()),
-  protocol: zod.string(),
-  receipt_mode: zod.number().nullable(),
-  roles: zod.object({}).passthrough(),
-  status: zod.number(),
-  team_id: zod.string().nullable().optional(),
-  type: zod.number(),
+  name: zod.string(),
```

Comment:
I commented out the fields we currently do not use for an export. 
But I want to leave them there, since finding the information what the data SHOULD contain is teadious